client/src/router/index.js

```diff
@@ -11,12 +19,43 @@ const routes = [
     component: Home
   },
   {
-    path: '/about',
-    name: 'about',
-    // route level code-splitting
-    // this generates a separate chunk (about.[hash].js) for this route
-    // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
+    path: '/settings',
+    name: 'SettingsPage',
```

Comment:
For main routes in views folder you can either create  a sub-folder for every route with index.vue file(similar to how things are done in Nuxt) OR simply make sure that route name equals to route component(for this project this is preferable - less confusion for students).

So please rename component to Settings - please make sure that main route components have the same name as their routes(for sub-routes this is situational - some main routes may have the same subroute names with different components). 

All main route-components must be placed into views folder - route-guards work only on route components - placing them here makes them easy to identify. This is also true for sub-route components.